cabal.project

```diff
@@ -3,12 +3,12 @@ packages: .
 source-repository-package
     type: git
     location: https://github.com/luc-tielen/llvm-codegen.git
-    tag: 42043c279797a4f57e254fcf5e60f3df56f9bf65
+    tag: c97de620ed147388cb6213864d33c63d199561e1
```

Comment:
Why are these pinned commits updated?

Reply:
Build was failing for me on MacOS as it was looking for LLVM-17 which is not available on my machine. The updated commit of llvm-codegen supports fallback to lower versions.

Reply:
Right, I forgot to bump that.. thank you :smile: